### DISCUSSION TITLE
Difference between "To Front" and "Bring Forward"

### DISCUSSION THREAD

**Discussion by ImperiaGP** [jgraph/drawio#2232] [Q&A] [answered]
As the title says: what is the difference between "To Front" and "Bring Forward" in the "Arrange" tab ?
They seem to do the same.

Idem ditto for "To Back" and "Send backward"

**Comment by davidjgraph** [comment, marked as answer]
To Front is bring all the way to the frontmost z-order value. Bring forward is bring forward beyond the single next cell currently in front of it in z-order (within its parent hierarchy, to be exact).
